docs/web/subgraph.mdx

```diff
@@ -15,7 +15,7 @@ This is a page covering the graph's ENS subgraph.
 
 ## The Graph
 
-The Graph is protocol for indexing and querying data from blockchains. There are multiple subgraphs that you can use to query information about ENS names.
+The Graph is a protocol for indexing and querying data from blockchains. There are multiple subgraphs that you can use to query information about ENS names.
 These subgraphs are available for [mainnet](https://api.thegraph.com/subgraphs/name/ensdomains/ens) and [goerli](https://api.thegraph.com/subgraphs/name/ensdomains/ensgoerli).
```

Comment:
Maybe also add Sepolia while you're here! https://api.studio.thegraph.com/query/49574/enssepolia/version/latest

Reply:
and holesky
[https://api.studio.thegraph.com/query/49574/ensholesky/version/latest⁠](https://api.studio.thegraph.com/query/49574/ensholesky/version/latest)

Reply:
Makes sense! Thanks for the links. I have added them in the latest commit.